src/api/translation_data.py

```diff
@@ -0,0 +1,11 @@
+from dataclasses import dataclass
+from typing import List, Tuple
+
+@dataclass
+class TranslationData:
```

Comment:
I like this, having it within a class allows us to update the result without changing the result type, however, it would be better to rename this to TranslationResult rather than TranslationData.